diff --git a/jhbuild/main.py b/jhbuild/main.py
index a5cf99b..3132d1d 100644
--- a/jhbuild/main.py
+++ b/jhbuild/main.py
@@ -94,9 +94,9 @@ def main(args):
         localedir = None
     gettext.install('jhbuild', localedir=localedir, unicode=True)
 
-    if hasattr(os, 'getuid') and os.getuid() == 0:
-        sys.stderr.write(_('You should not run jhbuild as root.\n').encode(_encoding, 'replace'))
-        sys.exit(1)
+#    if hasattr(os, 'getuid') and os.getuid() == 0:
+#        sys.stderr.write(_('You should not run jhbuild as root.\n').encode(_encoding, 'replace'))
+#        sys.exit(1)
 
     logging.getLogger().setLevel(logging.INFO)
     logging_handler = logging.StreamHandler()
diff --git a/jhbuild/utils/systeminstall.py b/jhbuild/utils/systeminstall.py
index 10e18d6..61218dc 100644
--- a/jhbuild/utils/systeminstall.py
+++ b/jhbuild/utils/systeminstall.py
@@ -467,9 +467,10 @@ class AptSystemInstall(SystemInstall):
 
         if native_packages:
             logging.info(_('Installing: %(pkgs)s') % {'pkgs': ' '.join(native_packages)})
-            args = self._root_command_prefix_args + ['apt-get', 'install']
+            args = self._root_command_prefix_args + ['apt-get', 'install', '-y', '--force-yes', '--allow-unauthenticated']
             args.extend(native_packages)
-            subprocess.check_call(args)
+            subprocess.check_call(args, env=dict(os.environ,
+                                  DEBIAN_FRONTEND="noninteractive"))
         else:
             logging.info(_('Nothing to install'))
 
