diff --git a/jhbuild/main.py b/jhbuild/main.py
index a5cf99b..3132d1d 100644
--- a/jhbuild/main.py
+++ b/jhbuild/main.py
@@ -94,9 +94,9 @@ def main(args):
         localedir = None
     gettext.install('jhbuild', localedir=localedir, unicode=True)
 
-    if hasattr(os, 'getuid') and os.getuid() == 0:
-        sys.stderr.write(_('You should not run jhbuild as root.\n').encode(_encoding, 'replace'))
-        sys.exit(1)
+#    if hasattr(os, 'getuid') and os.getuid() == 0:
+#        sys.stderr.write(_('You should not run jhbuild as root.\n').encode(_encoding, 'replace'))
+#        sys.exit(1)
 
     logging.getLogger().setLevel(logging.INFO)
     logging_handler = logging.StreamHandler()
